python-main.js

```diff
@@ -973,9 +973,9 @@ function web_editor(config) {
 
     function invalidFileWarning(fileType){
         if(fileType=="mpy"){
-            modalMsg(config['translate']['load']['invalid-file-title'],config['translate']['load']['mpy-warning'],"");
+            modalMsg(config['translate']['load']['invalid-file-title'],config['translate']['load']['mpy-warning'],"")
         }else{
-            modalMsg(config['translate']['load']['invalid-file-title'], config['translate']['load']['extension-warning'],"");
+            modalMsg(config['translate']['load']['invalid-file-title'], config['translate']['load']['extension-warning'],"")
```

Comment:
We can restore these two lines to the original with the semicolon at the end.

Reply:
fixed in https://github.com/bbcmicrobit/PythonEditor/pull/310/commits/7ebd2794a1f8ef0cd823a3ad67df4b139a80bdce